lib/matplotlib/mlab.py

```diff
@@ -69,6 +68,7 @@ def window_hanning(x):
     return np.hanning(len(x))*x
 
 
+@_api.deprecated("3.6", alternative="")
```

Comment:
Missing alternative.

Reply:
Its a no-op?  I guess I could say `lambda x: x`.

Reply:
`alternative=""` is the default. You don't have to specify this explicitly.

I'm fine either way: Don't give an alternative (we don't have to specify one). Or use `lambda x: x`.